net/tayga/src/opnsense/mvc/app/controllers/OPNsense/Tayga/forms/general.xml

```diff
@@ -0,0 +1,32 @@
+<form>
+    <field>
+        <id>general.enabled</id>
+        <label>Enable</label>
+        <type>checkbox</type>
+        <help>This will activate Tayga.</help>
+    </field>
+    <field>
+        <id>general.v4address</id>
+        <label>IPv4 Address</label>
+        <type>text</type>
+        <help>This is not your routers IPv4 address. Tayga requires its own address because it acts as an IPv4 and IPv6 router, and needs to be able to send ICMP messages. This address can safely be located inside the dynamic-pool prefix.</help>
```

Comment:
```suggestion
        <help>This is not your router's IPv4 address. Tayga requires its own address because it acts as an IPv4 and IPv6 router, and needs to be able to send ICMP messages. This address can safely be located inside the dynamic-pool prefix.</help>
```